server/controllers/match/placementRequests/occupancyViewController.test.ts

```diff
@@ -301,7 +301,7 @@ describe('OccupancyViewController', () => {
 
       const expectedDurationDays = 10
       const expectedStartDate = `${arrivalYear}-0${arrivalMonth}-${arrivalDay}`
-      const expectedParams = `apType=${apType}&startDate=${expectedStartDate}&durationDays=${expectedDurationDays}`
+      const expectedParams = `apType=${apType}&startDate=${expectedStartDate}&durationDays=${expectedDurationDays}&criteria=`
```

Comment:
If `criteria` is an empty string we shouldn't need to include this parameter at all

Reply:
True. It needs a tweak to the controller but a good shout